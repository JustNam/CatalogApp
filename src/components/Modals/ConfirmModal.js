import React, { Component } from 'react';
import { Modal, Button } from 'react-bootstrap';


export class ConfirmModal extends Component {
  render() {
    const { handleClose, handleConfirm } = this.props;
    return (
      <Modal
        animation={false}
        show={this.props.show}
        onHide={this.props.handleClose}
      >
        <Modal.Header closeButton><Modal.Title>Confirm</Modal.Title></Modal.Header>
        <Modal.Body>Are you sure you want to delete this item?</Modal.Body>
        <Modal.Footer>
          <Button variant="outline-success" size="sm" onClick={handleConfirm}>
            Yes
          </Button>
          <Button variant="outline-secondary" size="sm" onClick={handleClose}>
            Cancel
          </Button>
        </Modal.Footer>
      </Modal>
    );
  }
}
export default ConfirmModal;
