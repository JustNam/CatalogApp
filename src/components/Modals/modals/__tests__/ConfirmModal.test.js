import React from 'react';
import { shallow } from 'enzyme';
import ConfirmModal from 'components/Modals/ConfirmModal';

describe('CreateItemModal', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ConfirmModal />);
  });
  it('It should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});