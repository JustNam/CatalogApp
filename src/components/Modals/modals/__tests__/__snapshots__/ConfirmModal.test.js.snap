// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateItemModal It should render correctly 1`] = `ShallowWrapper {}`;
